styles/globals.scss

```diff
@@ -1,3 +1,2 @@
 @import "node_modules/govuk-frontend/govuk/all";
 @import "loader";
-@import "lincoln";
```

Comment:
Consider commenting this out instead to provide some level of sign posting to simplify future development of theme/skin.